client/component-lab/Banner/Banner.story.coffee

```diff
@@ -0,0 +1,58 @@
+React = require 'react'
+{ storiesOf, action } = require '@kadira/storybook'
+Button = require 'lab/Button'
+Box = require 'lab/Box'
+{ Header, Message, Footer, Container, List, Divider, Actions, PriceSegment } = Banner = require 'lab/Banner'
+styles = require './StoryStyles.stylus'
+
+storiesOf 'Banner', module
+
+
+  .add 'default', ->
+
+    <Banner
+      type="success"
+      image="banner_for_billing">
+      <Container>
+        <Header
+          heading="Boost Your Koding with"
+          target="Business Add-On and Support Plans"/>
+        <Message>
+          Fusce dapibus, tellus ac cursus commodo, tortor mauriscondimentum nibh, ut fermentum massa
+        </Message>
+        <Footer>
+          <a href=""><Button type="primary-1" size="medium">EXPLORE</Button></a>
+        </Footer>
+      </Container>
+    </Banner>
+
+
+  .add 'Business Add-On', ->
+
+    <Banner
+      type="primary"
```

Comment:
https://github.com/koding/styleguide-coffeescript#use-single-quotes-if-there-is-no-string-interpolation